cmds/core/exec.js

```diff
@@ -36,7 +36,7 @@ exports.builder = async (yargs) => {
 const EASK_HOMEDIR = os.homedir().replace(/\\/g, '/') + '/.eask/';
 
 exports.handler = async (argv) => {
-  process.env.EASK_HOMEDIR = EASK_HOMEDIR;  // setup environemnt, so Emacs can receive it
+  process.env.EASK_HOMEDIR = EASK_HOMEDIR; // setup environment, so Emacs can receive it
```

Comment:
This is personal references but I prefer 2 spaces after the code. Can you alter it back?